### DISCUSSION TITLE
Update go2rtc when using Frigate on Unraid?

### DISCUSSION THREAD

**Discussion by pdawg17** [blakeblackshear/frigate#7278] [Ask A Question] [answered]
I'm having some issues with go2rtc 1.2 so wanted to update to 1.5+. The instructions say to place the new file in /config but on Unraid I do not see a /config folder. Anyone here have the same setup and able to explain how to upgrade? Thanks!

**Comment by NickM-27** [comment, marked as answer]
On unraid `/mnt/user/appdata/frigate` is mapped to /config so that is where you would put the file

**Reply by pdawg17** [reply]
Ah so just drop the file in the frigate folder there? Thanks for the quick reply!

**Reply by NickM-27** [reply]
yes, though you need to make sure the entire /frigate folder is mapped and not just the /config.yml file

**Reply by pdawg17** [reply]
Thanks so much! Got it updated with your instructions.

**Comment by pdawg17** [comment]
> yes, though you need to make sure the entire /frigate folder is mapped and not just the /config.yml file

**Reply by MrDinte** [reply]
Don't quite understand this. I've added file to /frigate folder now there is 2 files config.yml and go2rtc is this correct?

**Reply by NickM-27** [reply]
yes, and you need to make sure the docker config maps the entire config directory

**Reply by MrDinte** [reply]
![image](https://github.com/blakeblackshear/frigate/assets/130459189/b481df32-edf5-4388-9d2b-dfe53af65251)
 like this?
sorry I only really use unraid for media server, this frigate and cctv is very new to me haha.

**Reply by NickM-27** [reply]
yes exactly

**Reply by MrDinte** [reply]
ok , 
![image](https://github.com/blakeblackshear/frigate/assets/130459189/536665c4-c498-418b-965c-7a83ab057592)

**Reply by MrDinte** [reply]
And just dump file here?

**Reply by NickM-27** [reply]
yes, and set its permissions to execute

**Reply by MrDinte** [reply]
Its set to RW RW RW is this not correct

**Reply by NickM-27** [reply]
that is only read/write not execute. You can run `chmod +X go2rtc` in the CLI to add this

**Reply by MrDinte** [reply]
Ok, I cant seem to get CLI to mnt my frigate?  mnt/user/appdata/frigate: No such file or directory

**Reply by NickM-27** [reply]
you need the trailing `/`

`cd /mnt/user/appdata/frigate`

**Reply by MrDinte** [reply]
thanks thats didn't work but i tried `chmod 755 go2rtc`

and now  
![image](https://github.com/blakeblackshear/frigate/assets/130459189/def3efcc-d502-4a5e-addb-8e895462ef1c)

**Reply by MrDinte** [reply]
![image](https://github.com/blakeblackshear/frigate/assets/130459189/df8ecc0a-85fb-4c60-b93d-46a9490eb6ad)

Thanks Mate, owe you a coffee
